mbtools/copy_resources_s3.py

```diff
@@ -23,6 +24,17 @@ def upload_resources(resource_dir, metadata_file, bucket, s3_dir):
         bucket, s3_dir, hashes_to_update, hash_to_filename_map, metadata_file
     )
 
+    if csv_file is not None:
+        output_hash_csv(hash_to_filename_map, csv_file)
+
+
+def output_hash_csv(hash_to_filename_map, csv_file):
+    with open(csv_file, "w") as f:
+        w = writer(f)
+        w.writerow(['hash', 'filename'])
```

Comment:
Minor: In so much as our intended use case is to copy / paste into [this file](https://docs.google.com/spreadsheets/d/1lCnVAJG9u6o_I7xC5vXLbKCskIZS0ez_qQ4zbGeYoRg), ordering `['filename', 'hash']` might simplify copy / pasting.